src/ape_ethereum/ecosystem.py

```diff
@@ -506,7 +506,7 @@ def str_to_slot(text):
                 # TODO perf: use a batch call here when ape adds support
                 storage = self.provider.get_storage(address, slot)
             except APINotImplementedError:
-                continue
+                break
```

Comment:
This seems like a pretty critical bugfix, can this be broken out and it's impact understood better?

Reply:
It isn't really that critical, but sure

Reply:
oh I see, it just exits early if this feature doesn't exist

Reply:
I still agree with the sentiment but it is just not that critical. Here is the PR: https://github.com/ApeWorX/ape/pull/2514

Reply:
yeah, I misunderstood what it was doing, I only saw the change in behavior and thought it was something bigger